src/constants.js

```diff
@@ -256,6 +256,13 @@ Constants.CompanyStatus = {
   NOT_ACTIVE: `isn't active`
 }
 
+Constants.TechnicalQualification = {
```

Comment:
I would prefer these constants to be grouped with the other Dynamics constants inside Constants.Dynamics (line 187 on)

Reply:
Changed as requested